props/blaster.h

```diff
@@ -233,7 +240,8 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
       next_action_ = NEXT_ACTION_NOTHING;
     }
   }
-  
+
+  uint32_t len;
```

Comment:
This name is too short for a global variable.
What is it for anyways?
Also, global variables should end in _.

Reply:
An errant remnant. Removed.